### PR TITLE
LRDOCS-8183 Document Previews Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-8183

@justinchoi001

### CODE REVIEW

File: docs/dxp/7.x/en/content-authoring-and-management/documents-and-media/uploading-and-managing/previewing-files.md [hunk 1]
```diff
@@ -1,3 +1,84 @@
...
+
+![Here is the comments section.](./previewing-files/images/03.png)
+
+### The Info Panel
+
+As mentioned above, clicking the *Info* icon (![Info](../../../images/icon-information.png)) opens the info panel. The top of the info panel displays the file's name, version, and [workflow status](../../../process-automation/workflow/user-guide/introduction-to-workflow.md).
+
+There are two tabs in the info panel: Details, and Versions. Details is selected by default and shows the following:
+
+![Here is the top half of Details section.](./previewing-files/images/04.png)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/content-authoring-and-management/documents-and-media/uploading-and-managing/previewing-files.md:42]: This image is egregiously large/tall. It has a very negative effect on readability. Please re-do this @justinchoi001.

File: docs/dxp/7.x/en/content-authoring-and-management/documents-and-media/uploading-and-managing/previewing-files.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+| --- | --- |
+| **Owner** | The file's owner. |
+| **Download** | A button to download the file. |
+| **Latest Version URL** | A URL to access the newest version of the file. |
+| **WebDAV URL** | A WebDAV URL for accessing the file via a desktop. |
+| **Document Type** | The file's document type. |
+| **Extension** | The file's extension. |
+| **Size** | The file's size on disk. |
+
+![Here is the bottom half of Details section.](./previewing-files/images/05.png)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/content-authoring-and-management/documents-and-media/uploading-and-managing/previewing-files.md:54]: This image is egregiously large/tall. It has a very negative effect on readability. Please re-do this @justinchoi001.

**Comment by justinchoi001** [comment]
I have removed the two images, combined the two tables, and renamed image 06.png as 04.png.
Is it possible to cherry pick this? https://github.com/justinchoi001/liferay-learn/commit/6511a896375e6a5c17ffa2ac373d61772bdd9d21

**Comment by jrhoun** [comment]
Merged. Thanks.

**Pull request closed without merging**